web/src/components/forms/validator-form.tsx

```diff
@@ -40,21 +45,43 @@ export const ValidatorForm: FC = () => {
   const [useExistingPEP, setUseExistingPEP] = useState(false);
   const [useExistingSchema, setUseExistingSchema] = useState(true);
   const [schemaString, setSchemaString] = useState<string | undefined>(undefined);
+  // const [activePepPasteTab, setActivePepPasteTab] = useState('pepPaste1');
 
   // watch the form data so we can use it
   const pepFiles = watch('pepFiles');
   const pepRegistryPath = watch('pepRegistryPath');
   const schemaFiles = watch('schemaFiles');
   const schemaRegistryPath = watch('schemaRegistryPath');
 
+  const [schemaPaste, setSchemaPaste] = useState<string>('');
+  // const [pep_Paste1, setPepPaste1] = useState<string>('');
+  // const [pep_Paste2, setPepPaste2] = useState<string>('');
+  
   const { data: schema } = useSchema(schemaRegistryPath?.value);
+  const params = useMemo(() => {
```

Comment:
Since this computation isn't expensive, we might be able to just directly create this variable.

```ts
let params = {}
if (useExistingPEP) {
  params = {
    pep: pepRegistryPath?.value,
        schema: schemaString,
        schema_registry: undefined,
        enabled: true,
      };
    } else {
      params = {
        pep: pepFiles,
        schema: schemaString,
        schema_registry: undefined,
        enabled: true,
      };
}
```

Reply:
Ah ok, we can do that modification when this is on dev